### ISSUE TITLE
[no-extra-spacing-attrs] Allow last space only on void tags 

### ISSUE THREAD

**Issue by gian1200** [yeonjuan/html-eslint#100]
Is it possible to enforce "no-extra-spacing-attrs" on all tags, except on those that are void tags?

In other words, make it work when using `"@html-eslint/require-closing-tags": ["error", { "selfClosing": "always" }]`

![image](https://user-images.githubusercontent.com/405347/183442441-58c1cb13-e194-4e8e-9bf1-6bb01b171fe8.png)

I found this other issue, bue am not sure if it's related or not https://github.com/yeonjuan/html-eslint/issues/76.

**Comment by yeonjuan** [comment]
Hi @gian1200 :) Thanks for the suggestion. Can you describe the option you suggest more?

it should ignore all other tags which except for void tags.

```html
<!-- no error ? -->
<div    id='aa'   class="class"></div>    
```

**Comment by gian1200** [comment]
Oh, my bad. I only meant to allow the last space, the one before "/>". All other current rules/directives should still apply.

**Comment by gian1200** [comment]
Hi, again.  First of all, thanks for tha awesome work you are doing with this plugin!

Maybe I missed something, I read the PR and updated my config file accordingly:

`"@html-eslint/no-extra-spacing-attrs": ["warn", { "enforceBeforeSelfClose": true }],`

However, I got the following error. Did I miss something?

![image](https://user-images.githubusercontent.com/405347/192925312-b75ed2e1-6f77-4012-b27f-aaa6fd3b32ee.png)
